README.md

```diff
@@ -10,19 +10,35 @@ At present `stellar-core-backup` makes a few assumptions about the environment t
 
 AWS credentials can be exported as environment variables or permissions can be granted by an IAM instance role.
 
+## GPG Backup Signing and Verification
+
+GPG verificaiton is enabled by default. Unless the user disables this securtiy feature with the --no-verify option, GPG signing and verification checks will be run.
+
+The appropriate GPG signing keys need to be in place and the key_id selection is configured in the configuration file.
+
+To install the public Stellar backup key for verification of Stellar provided backups, used the --getkey flag and the provided key-ID contained in the config/sample.yaml file. Change this value for your own key.
+
+`stellar-core-backup --getkey`
+
+The keyservers used to serve this key from belong to the SKS Keyserver service cluster hkp://pool.sks-keyservers.net. 
+
+If you try to verify a backup that has not been signed, you will generate an error.
+
+`gpg: can't open 'SHA256SUMS.sig': No such file or directory`.
+
 ## Configuration
 
 | config param | description |
 |--------------|-------------|
 |working_dir| Path to working directory which will hold temporary files, needs sufficient space to store and untar 1 backup|
 |core_config| Path to stellar-core configuration file of the node we are backing up, retrieves database credentials, etc.|
-|backup_dir| Path to directory which will hold the final backup|
 |s3_bucket| S3 bucket to store/retrieve buckets to/from|
 |s3_path| S3 Path prefix, can be used for backing up multiple core nodes to the same bucket|
+|gpg_key| GPG key ID used for signing and verification of the stellar-core backups. The provided ID is the Stellar signing key|
 
 ## Usage As Command Line Tool
 
-##### backup
+##### backup and shasum the backup files then gpg sign the resulting SHA256SUMS file
```

Comment:
This is a bit long, we have addressed that the backups are signed and verified unless the `--no-verify` flags is passed in the "GPG Backup Signing and Verification" section, do we need to reiterate this for the actual "backup" step ?